packages/flutter_test/lib/src/binding.dart

```diff
@@ -545,6 +551,13 @@ class LiveTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
   bool _inTest = false;
 
   @override
+  int get microtaskCount {
+    // Unsupported until we have a wrapper around the real async API (#4637)
```

Comment:
put a real hyperlink here to make it easier to get to the bug when you're reading the code